RELEASING.md

```diff
@@ -259,37 +278,37 @@ as described in the section above.
    make release-publish
    ```
 
-1. Merge the PR. This will cause candidate.docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct before proceeding to the next step.
-
-1. Checkout the master branch
-
-1. In [netlify.toml](netlify.toml):
-
-   1. Set the `CANDIDATE_RELEASE` environment variable back to an empty string.
-
-   1. Update the `CURRENT_RELEASE` environment variable.
+1. Merge the PR. 
 
-1. In [netlify/_redirects](netlify/_redirects), add a new line for the new release.
+1. On netlify update `Production branch` in Deploy contexts to `release-vX.Y` in docs.projectcalico.org site settings and trigger the deployment.
+This will cause docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct.
 
-1. Commit your changes. For example:
-
-   ```
-   git commit -m "Promote vX.Y.Z to latest"
-   ```
+## Adding the previous release to docs.projectcalico.org/archive
 
-1. Push your branch and open a pull request to the upstream master branch. Get it reviewed and wait for it to pass CI.
+1. Create a new site on netlify by linking previous release branch (ex: vY.Z) of this repository. Keep the default build settings. Rename the generated site to follow the same naming convention as other releases.
 
-## Adding the previous release to archive.docs.projectcalico.org
+1. Ensure deployment is done by visiting to `<site-name>.netlify.app/archive/vY.Z/`.
 
-1. Checkout latest master.
+1. Check out to production branch. This would be same branch from which `docs.projectcalico.org` root docs are generated.
```

Comment:
I don't think the average person understands what "root docs" means so maybe this wording will be more clear.
```suggestion
1. Check out to production branch. The production branch is constantly changing, but should be the current `release-vX.Y`.
```